griptape/processors/base_processors.py

```diff
@@ -0,0 +1,23 @@
+class BasePromptStackProcessor:
+    def before_run(self, prompt):
+        # Process the prompt before it is run
+        pass
+
+    def after_run(self, result):
+        # Process the result after the prompt is run
+        pass
+
+
+class BasePromptDriver:
```

Comment:
We've already got a [BasePromptDriver](https://github.com/griptape-ai/griptape/blob/main/griptape/drivers/prompt/base_prompt_driver.py), what is this?